libsolidity/formal/SymbolicState.h

```diff
@@ -171,6 +181,29 @@ class SymbolicState
 		},
 		m_context
 	};
+
+	BlockchainVariable m_crypto{
+		"crypto",
+		{
+			{"keccak256", std::make_shared<smtutil::ArraySort>(smt::smtSort(*TypeProvider::bytesStorage()), smtSort(*TypeProvider::fixedBytes(32)))},
+			{"sha256", std::make_shared<smtutil::ArraySort>(smt::smtSort(*TypeProvider::bytesStorage()), smtSort(*TypeProvider::fixedBytes(32)))},
+			{"ripemd160", std::make_shared<smtutil::ArraySort>(smt::smtSort(*TypeProvider::bytesStorage()), smtSort(*TypeProvider::fixedBytes(20)))},
```

Comment:
Maybe will increase readability, but not mandatory suggestion.
Also, would be great to be able to move this out of header somehow?

```suggestion
			{"keccak256", std::make_shared<smtutil::ArraySort>(
				smt::smtSort(*TypeProvider::bytesStorage()), 
				smtSort(*TypeProvider::fixedBytes(32))
			)},
			{"sha256", std::make_shared<smtutil::ArraySort>(
				smt::smtSort(*TypeProvider::bytesStorage()), 
				smtSort(*TypeProvider::fixedBytes(32))
			)},
			{"ripemd160", std::make_shared<smtutil::ArraySort>(
				smt::smtSort(*TypeProvider::bytesStorage()), 
				smtSort(*TypeProvider::fixedBytes(20))
			)},
```

Reply:
If I move it out of the header it needs to go in the initialization list. I personally find this way more readable than that.

Reply:
Indeed, let’s leave like this